Remove intl and locale support from the efibootdump

Copyright (C) 2017 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

diff --git a/src/efibootdump.c b/src/efibootdump.c
index 30a1943..2202432 100644
--- a/src/efibootdump.c
+++ b/src/efibootdump.c
@@ -13,8 +13,6 @@
 #include <efivar.h>
 #include <err.h>
 #include <inttypes.h>
-#include <libintl.h>
-#include <locale.h>
 #include <popt.h>
 #include <stdint.h>
 #include <stdio.h>
@@ -27,9 +25,9 @@
 
 int verbose;
 
-#define  _(String) gettext (String)
-#define Q_(String) dgettext (NULL, String)
-#define C_(Context,String) dgettext (Context,String)
+#define  _(String) String
+#define Q_(String) String
+#define C_(Context,String) dgettext String
 
 static void
 print_boot_entry(efi_load_option *loadopt, size_t data_size)
@@ -115,10 +113,6 @@ main(int argc, char *argv[])
 	char *guidstr = NULL;
 	efi_guid_t guid = efi_guid_global;
 
-	setlocale(LC_ALL, "");
-	bindtextdomain("efibootdump", LOCALEDIR);
-	textdomain("efibootdump");
-
 	struct poptOption options[] = {
 		{.argInfo = POPT_ARG_INTL_DOMAIN,
 		 .arg = "efibootdump" },
